### PR TITLE
feat!: 'decrypt_message' returns the sender client id [CL-81]

### PR DESCRIPTION
**Pull request by beltram**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Return sender client id when decrypting a message to help AVS

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: crypto/src/conversation/decrypt.rs [hunk 1]
```diff
@@ -799,5 +807,36 @@ pub mod tests {
             )
             .await
         }
+
+        #[apply(all_credential_types)]
+        #[wasm_bindgen_test]
+        pub async fn returns_sender_client_id(credential: CredentialSupplier) {
```
Comment (augustocdias) [inline crypto/src/conversation/decrypt.rs:813]: Is it possible to add a test for when it won't be present? Do we know which cases it will be none?
